src/components-examples/material/chips/chips-overview/chips-overview-example.html

```diff
@@ -1,6 +1,6 @@
-<mat-chip-list aria-label="Fish selection">
-  <mat-chip>One fish</mat-chip>
-  <mat-chip>Two fish</mat-chip>
-  <mat-chip color="primary" selected>Primary fish</mat-chip>
-  <mat-chip color="accent" selected>Accent fish</mat-chip>
-</mat-chip-list>
+<mat-chip-listbox aria-label="Fish selection">
```

Comment:
I think that this needs to be set as `multiple` since there are two selected options.

Reply:
I meant to only put `selected` on one of the chips.